src/scenic/core/simulators.py

```diff
@@ -621,6 +635,11 @@ def getReplay(self):
             raise RuntimeError('cannot save replay without replay support enabled')
         return self._replayOut.getBytes()
 
+    @classmethod
+    def extractEulerAngles(cls, global_orientation, parent_orientation):
+        return (global_orientation * parent_orientation.inverse).eulerAngles
```

Comment:
If this method is for the convenience of interface writers, it should have a docstring explaining when and how to use it.